doc/source/structures/collections/list.rst

```diff
@@ -50,33 +53,18 @@ Structure
         * - :meth:`REMOVE(index)`
           - None
           - remove item by index
-        * - :attr:`CLEAR`
+        * - :meth:`CLEAR()`
```

Comment:
Lists are the only thing that inherit from CollectionValue actually (currently).  Queue and Stack inherit directly from EnumerableValue.  So List is really the only structure where `Clear` shows up.